src/components/Header/Header.tsx

```diff
@@ -1,14 +1,27 @@
 import React from "react";
 import PetsIcon from "@mui/icons-material/Pets";
+import { makeStyles } from "@mui/styles";
+import { Theme } from "@mui/material";
+
+const useStyles = makeStyles((theme: Theme) => ({
+  SpalshDiv: {
+    display: "flex",
+  },
+  HeaderID: {
+    fontFamily: "Comfortaa,cursive",
+  },
+  PIcon: {
+    position: "relative",
+    marginTop: "7%",
+  },
+}));
+
 export default function Header() {
+  const HeaderClasses=useStyles();
```

Comment:
this should too in camelcase
```suggestion
  const headerClasses=useStyles();
```